ps_facetedsearch.php

```diff
@@ -60,15 +60,31 @@ public function __construct()
 
     public function install()
     {
-        if (parent::install() && $this->registerHook('leftColumn')
-        && $this->registerHook('categoryAddition') && $this->registerHook('categoryUpdate') && $this->registerHook('attributeGroupForm')
-        && $this->registerHook('afterSaveAttributeGroup') && $this->registerHook('afterDeleteAttributeGroup') && $this->registerHook('featureForm')
-        && $this->registerHook('afterDeleteFeature') && $this->registerHook('afterSaveFeature') && $this->registerHook('categoryDeletion')
-        && $this->registerHook('afterSaveProduct') && $this->registerHook('postProcessAttributeGroup')
-        && $this->registerHook('postProcessFeature') && $this->registerHook('featureValueForm') && $this->registerHook('postProcessFeatureValue')
-        && $this->registerHook('afterDeleteFeatureValue') && $this->registerHook('afterSaveFeatureValue') && $this->registerHook('attributeForm')
-        && $this->registerHook('postProcessAttribute') && $this->registerHook('afterDeleteAttribute') && $this->registerHook('afterSaveAttribute') && $this->registerHook('leftColumn')
-        && $this->registerHook('productSearchProvider')) {
+        if (
+            parent::install()
+            && $this->registerHook('categoryAddition')
+            && $this->registerHook('categoryUpdate')
+            && $this->registerHook('attributeGroupForm')
+            && $this->registerHook('afterSaveAttributeGroup')
+            && $this->registerHook('afterDeleteAttributeGroup')
+            && $this->registerHook('featureForm')
+            && $this->registerHook('afterDeleteFeature')
+            && $this->registerHook('afterSaveFeature')
+            && $this->registerHook('categoryDeletion')
+            && $this->registerHook('afterSaveProduct')
+            && $this->registerHook('postProcessAttributeGroup')
+            && $this->registerHook('postProcessFeature')
+            && $this->registerHook('featureValueForm')
+            && $this->registerHook('postProcessFeatureValue')
+            && $this->registerHook('afterDeleteFeatureValue')
+            && $this->registerHook('afterSaveFeatureValue')
+            && $this->registerHook('attributeForm')
+            && $this->registerHook('postProcessAttribute')
+            && $this->registerHook('afterDeleteAttribute')
+            && $this->registerHook('afterSaveAttribute')
+            && $this->registerHook('productSearchProvider')
+            && $this->registerHook('displayLeftColumn')
```

Comment:
For info, ``leftColumn`` was called twice, it's now renamed to ``displayLeftColumn``